### ISSUE TITLE
Policy to have two maintainers sign off code

### ISSUE THREAD

**Issue by untitaker** [pallets/meta#18]
Basically what the title says. Require two maintainers to give their :+1: to merge code. Not technically enforced, and maybe skippable for typo fixes.

**Comment by jeffwidman** [comment]
Hmm... I'd say I just screwed up today, missing things due to a combination of my inexperience and too much context switching in the midst of co-leading the Pycon sprint. So while I'm going to make it a general rule of thumb for myself that I shouldn't merge commits in the middle of a sprint, I'd hate to have my mistake result in a general policy for everyone else that slows things down.

Historically, my personal rule of thumb has been to get a second pair of eyes on code changes, but not on docs changes. My thought has been that if I occasionally miss something on a docs change, it's not hard to fix and there's no risk of accidentally introducing an actual code bug. I can be a bit more cautious on that front going forward. My only fear is that we risk leaving more PRs to languish in the queue because none of us ever actually pulled the trigger with a "yes" or "no"... IIRC, we've got a number of those already where the PR was commented on by a maintainer, fixed by the submitter, but then never merged or rejected.
